pkgs/top-level/aliases.nix

```
@@ -948,6 +948,7 @@ mapAliases ({
   tkcvs = tkrev; # Added 2022-03-07
   tokodon = plasma5Packages.tokodon;
   tor-browser-bundle-bin = tor-browser; # Added 2023-09-23
+  tootle = throw "'tootle' has been removed as it has been archived upstream. Consider using 'tuba' instead"; # Added 2024-01-31
```

The repo is actually moved to [gitlab.g.o](https://gitlab.gnome.org/World/tootle), though the new repo is also quite dead and the original author now actually [involves](https://gitlab.gnome.org/World/tootle/-/issues/1#note_1711833) in tuba :shrug: 

```suggestion
  tootle = throw "'tootle' has been removed as it is not maintained upstream. Consider using 'tuba' instead"; # Added 2024-02-11
  tor-browser-bundle-bin = tor-browser; # Added 2023-09-23
```